

fs/proc/proc_misc.c: In function `meminfo_read_proc':
fs/proc/proc_misc.c:235: warning: long unsigned int format, unsigned int arg (arg 20)
fs/proc/proc_misc.c:235: warning: long int format, int arg (arg 22)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/proc/proc_misc.c |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff -puN fs/proc/proc_misc.c~display-committed-memory-limit-and-available-in-meminfo-fix fs/proc/proc_misc.c
--- 25/fs/proc/proc_misc.c~display-committed-memory-limit-and-available-in-meminfo-fix	2004-09-30 23:28:31.034841496 -0700
+++ 25-akpm/fs/proc/proc_misc.c	2004-09-30 23:28:52.248616512 -0700
@@ -154,12 +154,14 @@ static int meminfo_read_proc(char *page,
 				 int count, int *eof, void *data)
 {
 	struct sysinfo i;
-	int len, committed, allowed;
+	int len;
 	struct page_state ps;
 	unsigned long inactive;
 	unsigned long active;
 	unsigned long free;
 	unsigned long vmtot;
+	unsigned long committed;
+	unsigned long allowed;
 	struct vmalloc_info vmi;
 
 	get_page_state(&ps);
@@ -202,8 +204,8 @@ static int meminfo_read_proc(char *page,
 		"Mapped:       %8lu kB\n"
 		"Slab:         %8lu kB\n"
 		"CommitLimit:  %8lu kB\n"
-		"Committed_AS: %8u kB\n"
-		"CommitAvail:  %8ld kB\n"
+		"Committed_AS: %8lu kB\n"
+		"CommitAvail:  %8lu kB\n"
 		"PageTables:   %8lu kB\n"
 		"VmallocTotal: %8lu kB\n"
 		"VmallocUsed:  %8lu kB\n"
_
